auto_nag/scripts/needinfo_regression_author.py

```diff
@@ -97,6 +97,16 @@ def bug_handler(regressor_bug):
         ).get_data().wait()
 
     def filter_bugs(self, bugs):
+        # Exclude bugs whose regressor author is nobody.
+        for bug in list(bugs.values()):
```

Comment:
```suggestion
        for bug in bugs.values():
```
As @calixteman suggested, I do not think `list()` is needed.

Reply:
The copy is needed to avoid changing the dictionary size while iterating over the dictionary.